dex/src/test/scala/com/wavesplatform/dex/api/ws/actors/WsExternalClientHandlerActorSpec.scala

```diff
@@ -20,7 +20,7 @@ import com.wavesplatform.dex.domain.account.KeyPair
 import com.wavesplatform.dex.domain.asset.Asset.{IssuedAsset, Waves}
 import com.wavesplatform.dex.domain.asset.AssetPair
 import com.wavesplatform.dex.domain.bytes.ByteStr
-import com.wavesplatform.dex.error.SubscriptionsLimitReached
+import com.wavesplatform.dex.error.{AddressAndPublicKeyAreIncompatible, InvalidJwtPayloadSignature, JwtBroken, JwtCommonError, JwtPayloadBroken, OrderAssetPairReversed, RequestArgumentInvalid, SubscriptionAuthTypeUnsupported, SubscriptionTokenExpired, SubscriptionsLimitReached, TokenNetworkUnexpected}
```

Comment:
Maybe, replace it with `import com.wavesplatform.dex.error._` ?

Reply:
+1

Reply:
resolved